Web/Edubase.Web.UI/Views/Downloads/Partials/_DownloadsResults.cshtml

```diff
@@ -47,12 +47,12 @@
 
         foreach (var group in downloadGroups)
         {
-            <h2 class="govuk-heading-m">@group.Heading</h2>
+            <h2 id="@StringUtils.ElementIdFormat(group.Heading)-heading" class="govuk-heading-m">@group.Heading</h2>
             <p>@group.Paragraph</p>
             foreach (var item in group.SubSections)
             {
                 <div class="govuk-form-group">
-                    <h3 class="govuk-heading-s">@item.Heading</h3>
+                    <h3 id="@StringUtils.ElementIdFormat(item.Heading)-heading" class="govuk-heading-s">@item.Heading</h3>
```

Comment:
I don't know it well enough so maybe a non-issue, but I note here we're two-loops deep.

Could `item.Heading` be repeated between groups?

Could `item.Heading` be the same value as `group.Heading`?

Suggestion, use both values within the nested part:

 ```cshtml
<h3 id="@StringUtils.ElementIdFormat(group.Heading)-@StringUtils.ElementIdFormat(item.Heading)-heading" class="govuk-heading-s">@item.Heading</h3>
```

Reply:
Hi @RogerHowellDfE thanks for spotting this.
I don't think we'd ever be in the position of this occurring as (see below) the group heading is `Establishment downloads` and the item heading is the sub group all of which have different values:
![image](https://user-images.githubusercontent.com/26785947/228176779-45236c68-e870-4faa-87ff-663d6181754b.png)